[2024-10-19T15:06:38.590+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: ambev_data_quality_pipeline.valido manual__2024-10-19T15:06:22.355408+00:00 [queued]>
[2024-10-19T15:06:38.597+0000] {taskinstance.py:1083} INFO - Dependencies all met for <TaskInstance: ambev_data_quality_pipeline.valido manual__2024-10-19T15:06:22.355408+00:00 [queued]>
[2024-10-19T15:06:38.597+0000] {taskinstance.py:1279} INFO - 
--------------------------------------------------------------------------------
[2024-10-19T15:06:38.597+0000] {taskinstance.py:1280} INFO - Starting attempt 1 of 1
[2024-10-19T15:06:38.598+0000] {taskinstance.py:1281} INFO - 
--------------------------------------------------------------------------------
[2024-10-19T15:06:38.606+0000] {taskinstance.py:1300} INFO - Executing <Task(BashOperator): valido> on 2024-10-19 15:06:22.355408+00:00
[2024-10-19T15:06:38.609+0000] {standard_task_runner.py:55} INFO - Started process 32159 to run task
[2024-10-19T15:06:38.611+0000] {standard_task_runner.py:82} INFO - Running: ['***', 'tasks', 'run', 'ambev_data_quality_pipeline', 'valido', 'manual__2024-10-19T15:06:22.355408+00:00', '--job-id', '50', '--raw', '--subdir', 'DAGS_FOLDER/dag_dataquality.py', '--cfg-path', '/tmp/tmpa54pxmfi']
[2024-10-19T15:06:38.612+0000] {standard_task_runner.py:83} INFO - Job 50: Subtask valido
[2024-10-19T15:06:38.623+0000] {logging_mixin.py:137} WARNING - /usr/local/lib/python3.9/site-packages/***/settings.py:249 DeprecationWarning: The sql_alchemy_conn option in [core] has been moved to the sql_alchemy_conn option in [database] - the old setting has been used, but please update your config.
[2024-10-19T15:06:38.656+0000] {task_command.py:388} INFO - Running <TaskInstance: ambev_data_quality_pipeline.valido manual__2024-10-19T15:06:22.355408+00:00 [running]> on host 39b0b1bd9d5a
[2024-10-19T15:06:38.700+0000] {taskinstance.py:1507} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=Klaus_Rezende
AIRFLOW_CTX_DAG_ID=ambev_data_quality_pipeline
AIRFLOW_CTX_TASK_ID=valido
AIRFLOW_CTX_EXECUTION_DATE=2024-10-19T15:06:22.355408+00:00
AIRFLOW_CTX_TRY_NUMBER=1
AIRFLOW_CTX_DAG_RUN_ID=manual__2024-10-19T15:06:22.355408+00:00
[2024-10-19T15:06:38.700+0000] {subprocess.py:63} INFO - Tmp dir root location: 
 /tmp
[2024-10-19T15:06:38.701+0000] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', "echo 'valido'"]
[2024-10-19T15:06:38.708+0000] {subprocess.py:86} INFO - Output:
[2024-10-19T15:06:38.708+0000] {subprocess.py:93} INFO - valido
[2024-10-19T15:06:38.709+0000] {subprocess.py:97} INFO - Command exited with return code 0
[2024-10-19T15:06:38.726+0000] {taskinstance.py:1318} INFO - Marking task as SUCCESS. dag_id=ambev_data_quality_pipeline, task_id=valido, execution_date=20241019T150622, start_date=20241019T150638, end_date=20241019T150638
[2024-10-19T15:06:38.743+0000] {local_task_job.py:208} INFO - Task exited with return code 0
[2024-10-19T15:06:38.756+0000] {taskinstance.py:2578} INFO - 0 downstream tasks scheduled from follow-on schedule check
